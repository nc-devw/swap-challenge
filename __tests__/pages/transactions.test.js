import { render } from "@testing-library/react";
import Transactions from "@/pages/transactions";

let transactions = [];

jest.mock("@/context/global", () => ({
  useAppContext: () => ({
    transactions,
  }),
}));

describe("Transaction", () => {
  it("should render correctly", () => {
    const { asFragment } = render(<Transactions />);
    expect(asFragment).toMatchSnapshot();
  });
});
