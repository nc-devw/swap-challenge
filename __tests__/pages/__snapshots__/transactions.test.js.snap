// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Transaction should render correctly 1`] = `
<DocumentFragment>
  <div
    class="container mx-auto"
  >
    <div
      class="rounded-lg m-12 border shadow bg-white undefined"
    >
      <div
        class="flex flex-col"
      >
        <div
          class="text-center p-4 border-b"
        >
          <p
            class="uppercase font-bold text-lg"
          >
            Transactions
          </p>
        </div>
        <div
          class="flex flex-col p-4 items-center justify-around border-b"
        >
          <span>
            There are no transactions
          </span>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
